packages/ui/src/memberships/components/MemberListItem/MyMember.tsx

```diff
@@ -47,7 +47,7 @@ export const MyMemberListItem = ({ member }: { member: Member }) => {
         <CountInfo count={invited} />
       </MemberColumn>
       <MemberControls>
-        <EditMembershipButton member={member} size="small">
+        <EditMembershipButton member={member} size="small" className="edit-button">
```

Comment:
Is this class used anywhere ?

Reply:
Yes, it is used for storybook test:
https://github.com/Joystream/pioneer/blob/53677ace201ab0324346ac6bdba35ce303f04ade/packages/ui/src/app/pages/Profile/MyMemberships.stories.tsx#L258C4-L258C73